src-core-fresh/fresh.iml

```diff
@@ -28,8 +28,9 @@ let incr_float () =
   count_float := Numeric.float_Add_6_6 !count_float (Numeric.Float_6 1);
   !count_float;;
 
+[@@@logic]
+
 let now_UTCTimestamp () = 
   Time_defaults_current_time.get_current_utctimestamp ()
+  [@@opaque]
```

Comment:
If you just want a single definition in logic mode you can use `[@@logic]`:
```suggestion
let now_UTCTimestamp () = 
  Time_defaults_current_time.get_current_utctimestamp ()
  [@@logic][@@opaque]
```

Reply:
ok thanks - will change to that

Reply:
actually, that doesn't work. I don't think we process any annotations (including `[@@logic]` in program mode)

Reply:
actually this doesn't quite work with the VSCode integration